src/metadata.ts

```diff
@@ -96,10 +97,20 @@ export interface ApiObjectMetadata {
 
   /**
    * Additional metadata attributes.
-   * @jsii ignore
-   * @see https://github.com/cdk8s-team/cdk8s-core/issues/1297
    */
-  readonly [key: string]: any;
+  readonly additionalAttributes?: { [key: string]: any };
```

Comment:
Breaking change to cleanup the `@jsii ignore` annotation and also make the code cleaner.